mne/minimum_norm/tests/test_inverse.py

```diff
@@ -425,19 +429,19 @@ def test_apply_inverse_operator(evoked):
     stc = apply_inverse(evoked, inverse_operator, lambda2, "sLORETA")
     assert stc.subject == 'sample'
     assert stc.data.min() > 0
-    assert stc.data.max() < 10.0
+    assert 2 < stc.data.max() < 5
```

Comment:
why did you need to update these tests?

Reply:
Didn't need to, but it helps during changes to the `inverse` code to have them be more precise